### DISCUSSION TITLE
AttributeError: module 'idom.html' has no attribute 'head'

### DISCUSSION THREAD

**Discussion by acivitillo** [idom-team/idom#572] [Problem] [answered]
There seems to be a bug in idom when using `idom.html.head`:

```python
@component
def Head():
    daisyui = html.link(
        {
            "href": "https://cdn.jsdelivr.net/npm/daisyui@1.22.0/dist/full.css",
            "rel": "stylesheet",
            "type": "text/css",
        }
    )
    return html.head(daisyui)
```

**Comment by acivitillo** [comment, marked as answer]
The documentation is not in line with the latest pip version. The `main` branch has these tags. However, the `main` branch is still missing `path` tag.

**Reply by rmorshea** [reply]
Created an issue for this: https://github.com/idom-team/idom/issues/576

**Reply by rmorshea** [reply]
In general, you can create any type of element you want using the `idom.vdom()` function. Just pass in `idom.vdom(tag, ...)` where `...` is all the same arguments from before.
